.github/workflows/post-merge-deploy-to-poc.yml

```diff
@@ -0,0 +1,65 @@
+name: Deploy to poc
+on:
+  push:
+    branches:
+      - OJ-892-deploy-common-lambdas-to-poc
```

Comment:
is this meant to be `main` for this PR?

Reply:
For now we are going to leave the workflow pointing to this branch, will update later when we have everything working

Reply:
@idrop the branch has now been amended to `main`